core/domain/certificate_assessment_services_test.py

```
@@ -195,6 +196,353 @@ def test_get_update_and_delete_certificate_assessment_offering(
+                    question_id_2: [self.topic_id],
+                    question_id_3: [self.topic_id],
+                },
+            },
+        ), mock.patch.object(
+            certificate_assessment_services,
+            'validate_certificate_assessment_offering',
+            return_value={'is_valid': True},
+        ), mock.patch.object(
+            certificate_assessment_services,
+            '_get_current_time',
+            return_value=datetime.datetime(2026, 8, 2, 0, 0),
```

Ruff (0.16.0)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)